operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -793,6 +794,13 @@ function generateMergedModel() {
 
   export __WLSDEPLOY_STORE_MODEL__="${NEW_MERGED_MODEL}"
 
+  local wdtArgs=""
+  wdtArgs+=" -oracle_home ${ORACLE_HOME}"
+  wdtArgs+=" ${model_list} ${archive_list} ${variable_list}"
+  wdtArgs+=" -domain_type ${WDT_DOMAIN_TYPE}"
+
+  trace "About to call '${WDT_BINDIR}/validateModel.sh ${wdtArgs}'."
+
   ${WDT_BINDIR}/validateModel.sh -oracle_home ${ORACLE_HOME} ${model_list} \
     ${archive_list} ${variable_list}  -domain_type ${WDT_DOMAIN_TYPE}  > ${WDT_OUTPUT} 2>&1
```

Comment:
If the call used the same 'wdtArgs' as the trace, then this would ensure they're in sync.